fixtures/datagen/fakerize.py

```diff
@@ -271,7 +284,7 @@ def make_person(vasp, verified=True, token="", rng=random.Random()):
     fake = Faker()
     name = fake.name()
     domain = rng.choice(DOMAINS)
-    email = name.lower().split()[0] + "@" + vasp.replace(" ", "").lower() + domain
+    email = name.lower().split()[0].split(".")[0] + "@" + vasp.replace(" ", "").lower() + domain
```

Comment:
Faker produces names with titles (Mr., Ms.) sometimes, this is a hacky workaround to make sure the email address is valid.